dts/bindings/radio_coex/sdc-radio-coex-three-wire.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+description: |
+    This is a representation of an external radio coexistence setup that has a
+    three-pin control interface (REQUEST, GRANT, PRIORITY/DIRECTION). This
+    extends that interface by adding additional parameters needed by the
+    Softdevice Controller's implementation.
```

Comment:
Not sure how I missed this before (maybe it got force-pushed over?), but here is a general note about writing binding descriptions.

Since DT is about describing hardware and its initial configuration, we try not to mention specific software consumers in places like this (cc @wiba-nordic as an FYI). There are obviously exceptions, but it's a good rule of thumb, and there are various reasons for it.

So something like this should probably be written to say something like "[...] by adding additional parameters needed to describe timing constraints and the direction of transmission" or so. That is, we prefer to describe things in terms of the hardware being described, not the software consuming the description.